pages/api/v1/answers.ts

```diff
@@ -769,16 +773,19 @@ async function storeActionsAwaitingConfirmation(
   }
 }
 
-async function streamResponseToUser(
-  res: ReadableStream<any>,
+export async function streamResponseToUser(
+  res: ReadableStream,
   streamInfo: (step: StreamingStepInput) => void,
-) {
+  valueVariableMap: Record<string, string>,
```

Comment:
This should be optional i think because you shouldn't need a "valueVariableMap" to stream a response to the user

Reply:
Ehh okay